base/applications/rapps/gui.cpp

```diff
@@ -598,13 +606,21 @@ CMainWindow::AddApplicationsToView(CAtlList<CAppInfo *> &List)
             m_ApplicationView->AddApplication(Info, bSelected);
         }
     }
+    m_ApplicationView->AddApplication(NULL, FALSE); // Tell the list we are done
 }
 
 VOID
 CMainWindow::UpdateApplicationsList(AppsCategories EnumType, BOOL bReload, BOOL bCheckAvailable)
 {
     bUpdating = TRUE;
 
+    if (HCURSOR hCursor = LoadCursor(NULL, IDC_APPSTARTING))
```

Comment:
What's the difference with `IDC_WAIT` ?
Wouldn't https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/include/reactos/ui/CWaitCursor.h;h=38bc74d614d59934030539d718e1bb7cd720d0bd;hb=HEAD
be of any use?

Reply:
IDC_WAIT is the big hourglass and IDC_APPSTARTING is the arrow with small hourglass and is less jarring when the wait is somewhat short.